src/components/LoginPage.js

```diff
@@ -345,7 +345,6 @@ const LoginPage = ({ theme }) => {
                   type="submit"
                   color="primary"
                   disabled={loading || !supportPassAuth}
-                  className={classes.button}
```

Comment:
I have not seen any reason for this. No class was defined for this on top of this file.